src/components/Strawpoll.tsx

```diff
@@ -82,12 +88,45 @@ function getNumberOfVotes(option: StrawpollOptionData, medium: StrawpollMedium)
   }
 }
 
+export function StrawpollConfirm(props: ModalProps) {
+  const onYesClick = () => {
+    props.onConfirm()
+    props.setOpen(false)
+  }
+
+  const onNoClick = () => {
+    props.setOpen(false)
+  }
+
+  return (
+    <Modal
+      size={"mini"}
+      centered={false}
+      open={props.open}
+      onClose={() => props.setOpen(false)}
+      onOpen={() => props.setOpen(true)}
+    >
```

Comment:
What happens if you pass the button as a `trigger` prop to `Modal` and use the `StrawpollModal` in the place of the button? Does it screw up the formatting of the button group?

Reply:
It does not! The format is a lot better that way so I've changed it to take the Button in as the trigger parameter.